doc/workshop/exercises/translation/reprojection-command-1.txt

```diff
@@ -1,4 +1,4 @@
-pdal translate ^
+pdal translate c:/Users/hobu/PDAL/exercises/analysis/ground/CSite1_orig-utm.laz ^
```

Comment:
In the beginning of the workshop, each time "C:/Users/hobu" is mentioned we remind them to change it to their folder. After a few occasions of this, we stop reminding them, presumably because it becomes redundant. I used "C:/Users/hobu...." to keep the workshop consistent with itself. I would agree that this could benefit from some clarification, however the materials needed to be printed this morning, so I made a PR for what I had completed.